sky/clouds/service_catalog/data_fetchers/fetch_aws.py

```diff
@@ -170,21 +166,24 @@ def _get_spot_pricing_table(region: str) -> pd.DataFrame:
     return df
 
 
-@ray.remote
 def _get_instance_types_df(region: str) -> Union[str, pd.DataFrame]:
     try:
         # Fetch the zone info first to make sure the account has access to the
         # region.
-        zone_df = ray.get(_get_availability_zones.remote(region))
+        zone_df = _get_availability_zones(region)
         if zone_df is None:
             raise RuntimeError(f'No access to region {region}')
 
-        df, offering_df, pricing_df, spot_pricing_df = ray.get([
-            _get_instance_types.remote(region),
-            _get_instance_type_offerings.remote(region),
-            _get_pricing_table.remote(region),
-            _get_spot_pricing_table.remote(region),
-        ])
+        with multiprocessing.pool.ThreadPool() as pool:
```

Comment:
Pool vs. ThreadPool? Use the same comment?
```
    # Use ThreadPool instead of Pool because this function can be called within
    # a multiprocessing.Pool, and Pool cannot be nested.
```